### ISSUE TITLE
1.14.0 breaks CommonJS imports

### ISSUE THREAD

**Issue by noomorph** [eta-dev/eta#210]
**Describe the bug**

Cannot run this code: `require('eta')`.

```
Error [ERR_REQUIRE_ESM]: require() of ES Module /Users/yaroslavs/Projects/wix/Detox/website/node_modules/eta/dist/eta.umd.js from /Users/yaroslavs/Projects/wix/Detox/website/[eval] not supported.
eta.umd.js is treated as an ES module file as it is a .js file whose nearest parent package.json contains "type": "module" which declares all .js files in that package scope as ES modules.
Instead rename eta.umd.js to end in .cjs, change the requiring code to use dynamic import() which is available in all CommonJS modules, or change "type": "module" to "type": "commonjs" in /Users/yaroslavs/Projects/wix/Detox/website/node_modules/eta/package.json to treat all .js files as CommonJS (using .mjs for all ES modules instead).
```

**To Reproduce**

```
mkdir repro
cd repro
npm init -y
npm install eta --save-dev
node -p "require('eta')"
```

**Expected behavior**

It should run without errors and return an object, like the version `1.13.0`.

**Screenshots**

| eta@1.13.0 | eta@1.14.0 |
| ------ | ------ |
| ![before](https://user-images.githubusercontent.com/1962469/213862396-95dfd83a-7129-4f08-9576-f6a4fb7de4b1.png) | ![after](https://user-images.githubusercontent.com/1962469/213862424-4ff47828-9e1d-465e-8dde-b93e300725a1.png) |

**Package & Environment Details**

 - Environment: Node.js (14, 16, 18)
 - Version: eta@1.14.0

**Additional context**

There was a similar issue here: https://github.com/nebrelbug/npm-to-yarn/issues/30

See how it was resolved: https://github.com/nebrelbug/npm-to-yarn/commit/4a13cc89ee13b16a578df08351ca0478580410bc

**Comment by gurgunday** [comment]
This is an easy fix and a possible remedy is to stay at `1.13`. Sorry for the problem – update will be released ASAP!

**Comment by nebrelbug** [comment]
@all-contributors please add @gurgunday for code

**Comment by nebrelbug** [comment]
@noomorph we just released a new patch version, 1.14.1, which should solve the issue! Let us know if you continue to have problems!
